s2fft/precompute_transforms/construct.py

```diff
@@ -193,39 +369,103 @@ def wigner_kernel(
         forward (bool, optional): Whether to provide forward or inverse shift.
             Defaults to False.
 
+        mode (str, optional): Whether to use FFT approach or manually compute each element.
```

Comment:
Mode is good here since we're not selecting recursion but the underlying compute mode.  However, let's switch `manual` to `direct` (`manual` suggests some kind of manual computation to me rather than just computing this directly, which I think is also the terminology sometimes used elsewhere).

Reply:
Changed to direct.